examples/producer.c

```diff
@@ -225,7 +210,7 @@ int main (int argc, char **argv) {
                 } else {
                         fprintf(stderr, "%% Enqueued message (%zd bytes) "
                                 "for topic %s\n",
-                                len, rd_kafka_topic_name(rkt));
+                                len, topic);
                 }
 
 
```

Comment:
You could also use the deliver report event to serve up your reports from the background queue no? We should provide an example of that.

Reply:
These are aimed to the be bare minimal examples, there's a bunch of existing more complex examples.

As for background thread delivery reports; another day :)